interface/main/tabs/main.php

```diff
@@ -331,7 +331,7 @@ function setupI18n(lang_id) {
 </style>
 
 </head>
-<body data-bind="css: responsiveDisplay.objWidth().bodyMain">
+<body data-bind="css: responsiveDisplay.objWidth().bodyMain" style="width: max-content;min-width: 100%;">
```

Comment:
`max-content` is not supported in IE and `min-width: 100%` can be replaced with bs4 `mw-100`

Reply:
Thanks for the review @im-Amitto

Reply:
Btw @bradymiller is OpenEMR is supported in IE
![image](https://user-images.githubusercontent.com/47683556/76343789-9fda1c80-6326-11ea-84de-e6f7d5db3fae.png)
I thought to test my changes and saw few million bugs 🤔

Reply:
This is a good question:
@sjpadgett , @tywrenn ; I recall this was discussed before. What IE version compatibility are we now shooting for?

Reply:
I was on IE 11 and btw new Edge chromium has no compatiblity issue 👍

Reply:
@bradymiller @im-Amitto Internet Explorer is a dead browser in which compatibility is basically no longer possible with a lot of frameworks outright not supporting it. It also is actually now considered a security risk as it is not really maintained by Microsoft and is expected to be removed in the future. Therefore this next OpenEMR revision is basically not gonna support IE and should not support it.

Reply:
RIP Internet Explorer. But i will never miss you🤣

Reply:
farewell my `not` friend, you will be truly `not` missed :)

Reply:
mw-width is replacement for max-width so can't go for it 😞. Should i go for min-vw-100 🤔

Reply:
@yashrajbothra I don't think min-vw-100 even exists in BS4 lol

Reply:
https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-viewport

Reply:
@yashrajbothra guess I learn something new everyday 😂

Reply:
@yashrajbothra  Sorry, I read it `max-width`. I guess i need to re-stock my caffine :confounded: